target/linux/ramips/dts/RB750Gr3.dts

```diff
@@ -97,12 +109,13 @@
 &ethernet {
 	mtd-mac-address = <&factory 0xe000>;
 	mtd-mac-address-increment = <1>;
+	ralink,port-map = "wllll";
```

Comment:
Does this make the rb750gr3 setting in target/linux/ramips/base-files/etc/board.d/02_network obsolete? Means, are the vlans set by driver according to the portmap. Please test this with an clean image. Do not preserve the config when doing an sysupgrade.